tasks/pipeline.py

```diff
@@ -134,17 +135,20 @@ def workflow_rules(gitlab_file=".gitlab-ci.yml"):
 
 
 @task
-def trigger(
-    _, git_ref=DEFAULT_BRANCH, release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"
-):
+def trigger(_, git_ref=DEFAULT_BRANCH, release_version_6="dev", release_version_7="dev-a7", repo_branch="dev"):
```

Comment:
This looks like it's reverting a linter change in addition of the new default parameter, doesn't this introduce a new warning/complaint?

Reply:
I did merge main but should've rebase instead will rebase to not add regression

Reply:
Actually no it's good because `nightly` is longer then `dev` that's why before we had the linter change to not exceed the maximum column size